app/src/api/learningJourney.js

```diff
@@ -62,19 +67,20 @@ export const createLearningJourneyWithJobId = async (jobId) => {
 };
 
 export const deleteLJWithLJId = async (ljId) => {
-  const deleteLJCourseRes = deleteLearningJourneyCourseWithLJId(ljId.LJId);
-  if (deleteLJCourseRes) {
-    try {
-      const deleteLJRes = await axiosLJInstance.delete(`/${ljId.LJId}`);
-      if (!deleteLJRes) {
-        throw new Error("No data returned from backend");
-      }
+  try {
+    const [deleteLJCourseRes, deleteLJRes] = await Promise.all([
+      await axiosLJCourseInstance.delete(`all/${ljId.LJId}`),
```

Comment:
\nit I'm thinking this might be clearer if we have it in the axiosInstance body. I think there is a way to do it for axios.